From 6299b644689b751abb970b5d9fbac068de7cbf3e Mon Sep 17 00:00:00 2001
From: Rob Herring <ra7055@freescale.com>
Date: Tue, 22 Jan 2008 15:29:53 -0600
Subject: [PATCH] ENGR00058219 mx31 make mach kconfig default no

MX31 Make board/machine kconfig default to no.

This fixes build break for MX31ADS.
---
 arch/arm/mach-mx3/Kconfig |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx3/Kconfig b/arch/arm/mach-mx3/Kconfig
index e06067c..cc52847 100644
--- a/arch/arm/mach-mx3/Kconfig
+++ b/arch/arm/mach-mx3/Kconfig
@@ -3,14 +3,12 @@ menu "MX3 Options"
 
 config MACH_MX31ADS
 	bool "Support MX31ADS platforms"
-	default y
 	help
 	  Include support for MX31ADS platform. This includes specific
 	  configurations for the board and its peripherals.
 
 config MACH_MX31_3DS
 	bool "Support MX31/MX32 3-Stack platforms"
-	default y
 	help
 	  Include support for MX31/MX32 3-Stack platform. This includes specific
 	  configurations for the board and its peripherals.
-- 
1.5.4.4

